.github/workflows/tests-rdbstorage.yml

```diff
@@ -0,0 +1,103 @@
+name: Tests (RDB storage)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+  schedule:
+    - cron: '0 23 * * SUN-THU'
```

Comment:
is this schedule is expected?

Reply:
I think this schedule is appropriate. The test is executed every weekday. Do you have any concerns?

Reply:
No concerns! Thanks.